interfaces/Portalicious/src/app/components/form/form-sidebar.component.html

```diff
@@ -1,7 +1,7 @@
 <p-drawer
   #sideBar
   [(visible)]="visible"
-  position="right"
+  [position]="rtlHelper.createPosition('right')()"
```

Comment:
We could have the Helper-service act as a sort-of polyfill, so that we provide it with "start" and that it outputs "left" or "right", depending on the `isRtl`-state...

But maybe its better to leave the "buggy PrimeNG-implementation" in-place, to highlight the need for: https://github.com/orgs/primefaces/discussions/3649
(Or even create a PR for that issue ourselves...)